### DISCUSSION TITLE
display all items

### DISCUSSION THREAD

**Discussion by NFLD99** [shedaniel/RoughlyEnoughItems#872] [Q & A] [answered]
Q: in REI how can i make it display everything as right now it hides things like facades in AE2

**Comment by shedaniel** [comment, marked as answer]
Is AE2 hiding the items by default? If so you might want to ask support for an option from AE2.

Otherwise, it would be configured by the REI filtering screen, which facades are hidden by modpacks usually, you can scroll to the bottom in the options screen to get to the filtering screen.
